### ISSUE TITLE
Creating database fails on `hobby-0`

### ISSUE THREAD

**Issue by henryoswald** [CrunchyData/terraform-provider-crunchybridge#2]
It is not currently possible to create a cluster on the hobby-0 plan as the memory value is returned as float and is parsed as an int.

```
Error: error unmarshaling response body (cluster detail): json: cannot unmarshal number 0.5 into Go struct field ClusterDetail.memory of type int
```

**Comment by wilybrace** [comment]
@henryoswald Sorry for missing the issue/PR, this is being incorporated into the next update we publish, likely the coming week.

**Comment by wilybrace** [comment]
Resolved in release v0.2.0, available via Terraform registry.
